### PR TITLE
Adding basic postgres configuration

### PR DESCRIPTION
**Pull request by edisinovcic**
Solves: https://github.com/ethereum/source-verify/issues/111
Adding basic configuration for running postgres in docker.

### CODE REVIEW

**Comment by edisinovcic** [comment]
@cgewecke @chriseth I'm not sure if this is to much indexes or are we missing some?
Also please check that types are correct.

**Comment by cgewecke** [comment]
@edisinovcic Are addresses part of this DB?

**Comment by edisinovcic** [comment]
> @edisinovcic Are addresses part of this DB?

I forgot the most important part :)

Will add it now.

**Comment by edisinovcic** [comment]
Try blob instead of text.

**Comment by edisinovcic** [comment]
INDEX on chain_id and address.

**Comment by edisinovcic** [comment]
INDEX address

**Comment by edisinovcic** [comment]
INDEX on chain_id, deployed_code_hash, creation_bytecode
INDEX on address, deployed_code_hash, creation_bytecode

File: src/postgres/schema.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE bytecode (
+    id uuid PRIMARY KEY,
+    address text NOT NULL,
+    creation_bytecode text NOT NULL,
```
Comment (chriseth) [inline src/postgres/schema.sql:6]: Can we make these binary? I think text is quite a waste of space.
It has to allow for zero-bytes, though.

File: src/postgres/schema.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    chain_id integer NOT NULL,
+    block_number integer NOT NULL,
```
Comment (chriseth) [inline src/postgres/schema.sql:8]: Ah if we add the block number (which is useful information) we have to check to select the largest one for querying the current deployed bytecode hash.

Comment (chriseth) [reply src/postgres/schema.sql:8]: Ah sorry, there is another primary key anyway.

File: src/postgres/schema.sql (hunk 1 above)
Comment (edisinovcic) [reply src/postgres/schema.sql:6]: Probably bitea format would be best. 

https://www.postgresql.org/docs/9.0/datatype-binary.html

File: src/postgres/schema.sql (hunk 2 above)
Comment (edisinovcic) [reply src/postgres/schema.sql:8]: We talked about storing all the changes from oldest to latest, but to show users only the latest change. In the future if we want to track changes in time we could easily support this and you could track changed for a contract in time.

**Comment by edisinovcic** [comment]
Adding db seeder: https://github.com/Shard-Labs/source-verify-seeder

**Comment by edisinovcic** [comment]
SQL lite will be used.

**Pull request closed without merging**